include/coap2/coap_net_internal.h

```diff
@@ -341,5 +341,7 @@ coap_pdu_t *coap_wellknown_response(coap_context_t *context,
  */
 unsigned int coap_calc_timeout(coap_session_t *session, unsigned char r);
 
+/** @} */
```

Comment:
This one is already in commit 1b15bf24 (sorry, for interfering).

Reply:
No worries - rebase has cleared things up.  rebase code pushed.

Reply:
Btw: I get a weird doxygen warning for `coap_dtls_new_context()` which will lead to build errors once this PR is merged. I have already spent some time trying to understand why doxygen thinks that `coap_context` would have multiple param sections:

```
/home/obgm/develop/libcoap/include/coap2/coap_dtls.h:479: warning: argument 'coap_context' from the argument list of coap_dtls_new_context has multiple @param documentation sections
```

Reply:
Hmm - strange.  Not seeing this (or can see why) on my Ubuntu-20 system.  Looking into it further.

Reply:
Could it be the later Doxyfile.in `RESOLVE_UNNAMED_PARAMS = YES` (not available in Ubuntu 20) that is causing the issue?

Reply:
As far as I can see there is nothing in the Doxyfile that causes this effect. Trial and error did not yet yield any useful results (except that I can delete the doxygen comment to make the warning go away.) There was [doxygen issue 6754](https://github.com/doxygen/doxygen/issues/6754) that talks about false positives in the parameter list but this should have been fixed in release 1.9.1 which I am currently using.
Trying more...

Reply:
Doxygen got confused by the preceding defgroup start. Commit 9e3ba50 now fixes the issue.